#! /usr/bin/env python

PACKAGE='pcl_object_clustering'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()
#       Name       Type      Level Description     Default Min   Max
gen.add("sac_dist_threshold",   double_t,   0, "SAC segmentation distance threshold", 0.03, 0.01, 0.1)
gen.add("ec_cluster_tolerance", double_t,   0, "Euclidean cluster tolerance", 0.05, 0.01, 0.1)
gen.add("ec_min_size",          int_t,      0, "min cluster size", 100, 2, 1000)
gen.add("ec_max_size",          int_t,      0, "max cluster size", 2000, 2, 4000)
exit(gen.generate(PACKAGE, "pcl_object_clustering", "PclObjectClustering"))